backend/src/authentication/user.rs

```diff
@@ -366,6 +366,21 @@ pub async fn list(conn: impl DbConnLike<'_>) -> Result<Vec<User>, Error> {
     Ok(users)
 }
 
+/// Fetch the first admin user ever created
+pub async fn get_active_user_count(conn: impl DbConnLike<'_>) -> Result<u32, AuthenticationError> {
+    let row = sqlx::query!(
+        r#"
+        SELECT COUNT(*) AS "count: u32"
```

Comment:
Do we really need the active user count? For the current endpoint `SELECT 1 FROM users WHERE last_activity_at IS NOT NULL LIMIT 1` with a `fetch_optional().await?.unwrap_or(0)` would also work and prevent any unnecessary work of scanning the users table (not that the users table is expected to be particularly large though.

Reply:
See [8015bdf](https://github.com/kiesraad/abacus/pull/1911/commits/8015bdff34bfb8d5adba97f5c68a3ab8d4ae232c)